client/src/components/Review.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Review = (props) => {
+  const {
+    ID, STARS, REVIEW_DATE, REVIEW_TEXT, USEFUL, FUNNY, COOL, user,
+  } = props.review;
+  // const STARS = props.review.STARS;
+  // const REVIEW_DATE = props.review.REVIEW_DATE;
+  // const REVIEW_TEXT = props.review.REVIEW_TEXT;
+  // const USEFUL = props.review.USEFUL;
+  // const FUNNY = props.review.FUNNY;
+  // const COOL = props.review.COOL;
+  // const user = props.review.user;
```

Comment:
Are these commented out lines needed any longer?

Reply:
updated in latest commit